services/test-results.js

```diff
@@ -84,8 +84,31 @@ function renderTestResultBadge({
   return { message, color }
 }
 
+const getDocumentation = ({ route }) => `
```

Comment:
This is one of those things that I saw duplicated in [appveyor-tests.service.js](https://github.com/badges/shields/blob/master/services/appveyor/appveyor-tests.service.js#L9) and [azure-devops-tests.service.js](https://github.com/badges/shields/blob/master/services/azure-devops/azure-devops-tests.service.js#L32). 

Instead of replicating it again for JenkinsTests, I figured it would be better to put it here in one place and consume it from the other services (if there's no opposition to this, I will open a future PR to update Appveyor and Azure DevOps test services to leverage this). We've also got some additional test badges that have been requested (for example CircleCI and Sonar) that can re-use this as well.

Reply:
Sounds good! I'm not sure it's essential to put the route in there, and leaving it out would simplify this, though I don't object.

Reply:
Yeah I think having the full route in the doc was more helpful back before the window we have today that will auto populate the the query params for users. 

I can remove that section (and change this to a simple static prop) if we're good with no longer including that